package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.1.6",
+  "version": "3.2.0",
```

Comment:
Excellent.

While you're here, I think it would be good to update sbc-common-components to the latest version, which supports account id sync. See https://github.com/bcgov/bcregistry/pull/51 for details, or we can chat about it some time.

Reply:
Absolutely, will do this!

Reply:
So mostly just unit tests breaking, but will work through these 👍

Reply:
I'm a bit of a loss how to resolve these failing tests 😵 
Will keep testing but it's quite odd, as when run individually they can succeed but when run in the full test suite, they fail and/or time out. 
Interesting problem for sure, not one i think we can ignore since we will NEED to update the common components.